cpp/cmake/Modules/ConfigureCUDA.cmake

```diff
@@ -43,5 +43,16 @@ endif()
 # Debug options
 if(CMAKE_BUILD_TYPE MATCHES Debug)
   message(VERBOSE "CUDF: Building with debugging flags")
+  list(APPEND CUDF_CXX_FLAGS -rdynamic)
   list(APPEND CUDF_CUDA_FLAGS -Xcompiler=-rdynamic)
 endif()
+
+if(CUDF_BUILD_STACKTRACE_DEBUG)
+  message(VERBOSE "CUDF: Building with flags '-rdynamic -Og -NDEBUG'")
+  set(CMAKE_CXX_FLAGS_RELEASE "") # Remove the current optimization level
+  set(CMAKE_CUDA_FLAGS_RELEASE "") # Remove the current optimization level
```

Comment:
Are there other flags not related to optimization that this could be overwriting?